testdata/project-v4-multigroup/cmd/main.go

```diff
@@ -287,54 +287,54 @@ func main() {
 	}
 	// nolint:goconst
 	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
-		if err = webhookshipv2alpha1.SetupCruiserWebhookWithManager(mgr); err != nil {
+		if err := webhookshipv2alpha1.SetupCruiserWebhookWithManager(mgr); err != nil {
 			setupLog.Error(err, "unable to create webhook", "webhook", "Cruiser")
 			os.Exit(1)
 		}
 	}
-	if err = (&seacreaturescontroller.KrakenReconciler{
+	if err := (&seacreaturescontroller.KrakenReconciler{
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "Kraken")
 		os.Exit(1)
 	}
-	if err = (&seacreaturescontroller.LeviathanReconciler{
+	if err := (&seacreaturescontroller.LeviathanReconciler{
```

Comment:
Why would we need to add := for all errors?
Why it would be required ?
What problem did you face that is solved within?

Reply:
I don't think we must necessarily be unified with `err :=`.
I have submitted this PR as a refactor.

Since `if err :=` and `if err =` were mixed in `cmd/main.go` generated by kubebuilder, I thought it would be better to unify one or the other.